### DISCUSSION TITLE
HORARIOS MATERIA. PRESENCIALIDAD/VIRTUALIDAD

### DISCUSSION THREAD

**Discussion by malenarmr** [INGCOM-UNRN-P1/cursada-2023#50] [General de la cátedra]
Holi a todos. 
Quería preguntar (a profes y compas) como es el tema de la organización de los horarios. 
Yo Estoy en la comisión 1 de Ivan, el horarios es Martes 16:30 y miércoles 16:00, el profe Ivan me dijo por mail que los martes iban a transmitir las clases y el profe Martín dijo que solo se van a transmitir la de los miércoles. 

Cuestión, en mi caso preferiría hacer todo lo posible virtual e ir cuando sea necesarios (vivo en La angostura y es un temita el transporte). 
Hay chance de hacer toda la materia virtual? como se vienen manejando? Se va a transmitir una sola clase de la materia? aiuda

**Comment by martinvilu** [comment]
@inomdedeu no está esta semana, esta @miguelmariguin en su lugar.

Lo que te comente tiene que ver con que no hay conexión como para hacer transmisión en vivo de la clase, en Anasagasti y por lo menos en los horarios puntuales en los que estoy. Lo único que se puede hacer es grabar y compartir.

Esto para ajustar las expectativas con respecto a la transmisión los días en los que estamos físicamente en las aulas de la universidad.
